README.md

```diff
@@ -1 +1,194 @@
-# Capstone-Project-02
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ OPTIONAL] Please make sure you remove this microverse logo from your readme. It is meant to be replaced with your project logo if you have one. :pray: